### PR TITLE
keepassxc: as an option do not build documentation

### PR DESCRIPTION
**Pull request by cessenat**
This avoids the dependency to asciidoctor and thus to ruby.

### CODE REVIEW

**Comment by cessenat** [comment]
Note for reviewer : as a default, documentation is built - as is the case for linux distribution packages - but I'm not sure it is a good idea since most spack packages as a default do not build the documentation. What do you recommend ?

File: var/spack/repos/builtin/packages/keepassxc/package.py [hunk 1]
```diff
@@ -68,6 +69,10 @@ def cmake_args(self):
             args.append("-DWITH_XC_ALL=ON")
         else:
             args.append("-DWITH_XC_ALL=OFF")
+        if "+docs" in spec:
+            args.append("-DWITH_XC_DOCS=ON")
+        else:
+            args.append("-DWITH_XC_DOCS=OFF")
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/keepassxc/package.py]:
```suggestion
        args.append(self.define_from_variant("WITH_XC_DOCS", "doc"))
```

Comment (alalazo) [reply var/spack/repos/builtin/packages/keepassxc/package.py]: Can also be applied elsewhere in the recipe.

**Review by alalazo — CHANGES_REQUESTED** [review]

**Comment by alalazo** [comment]
@cessenat Can you also take care of:

> Can also be applied elsewhere in the recipe.

? Should be just the other variant above your modification.

**Comment by cessenat** [comment]
Oh yes, thanks.
BTW the doc option I chose is "docs" as for cmake WITH_XC_DOCS.
Some package use doc other docs...

**Comment by alalazo** [comment]
> Some package use doc other docs...

True, we don't have a de facto standard for that - so I guess it's up to you :wink:

**Review by alalazo — APPROVED** [review]

**Comment by cessenat** [comment]
Hi @alalazo can you merge to develop ?
Same for PR #32032 ?
Thanks.

**Pull request merged**